pkg/controller/certificates/signer/signer.go

```diff
@@ -185,17 +185,18 @@ func (s *signer) handle(csr *capi.CertificateSigningRequest) error {
 	return nil
 }
 
-func (s *signer) sign(x509cr *x509.CertificateRequest, usages []capi.KeyUsage, now func() time.Time) ([]byte, error) {
+func (s *signer) sign(x509cr *x509.CertificateRequest, usages []capi.KeyUsage, expirationSeconds *int32, now func() time.Time) ([]byte, error) {
 	currCA, err := s.caProvider.currentCA()
 	if err != nil {
 		return nil, err
 	}
 	der, err := currCA.Sign(x509cr.Raw, authority.PermissiveSigningPolicy{
-		TTL:      s.certTTL,
-		Usages:   usages,
-		Backdate: 5 * time.Minute, // this must always be less than the minimum TTL requested by a user
-		Short:    8 * time.Hour,   // 5 minutes of backdating is roughly 1% of 8 hours
-		Now:      now,
+		TTL:               s.certTTL,
```

Comment:
update docs on certTTL to indicate this is the max TTL and that individual requests might request shorter certs